meow/frontend/src/components/examples/PostMaker.js

```diff
@@ -1,15 +1,15 @@
-import React from "react";
-import axios from "axios";
+import React from 'react';
+import axios from 'axios';
 
-const ENDPOINT = "/api/post/";
+const ENDPOINT = '/api/post/';
 
 export default class PostMaker extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
```

Comment:
we should use [class fields](https://github.com/tc39/proposal-class-fields) instead of constructors. @dustinnewman98 does our current webpack config support?